airflow/_vendor/slugify/slugify.py

```diff
@@ -1,3 +1,6 @@
+# -*- coding: utf-8 -*-
+# pylint: skip-file
+"""Slugify !"""
```

Comment:
I'd rather we don't touch the files under airflow/_vendor/ unless we _have_ to.

Reply:
(Oh this change is probably from one of the other PRs isn't it? So that comment applies somewhere else)

Reply:
Yeah. I will split those and exclude vendor from the original change. I thought I did that everywhere but I might have corrected vendor accidentally.

Reply:
Removed in the first commit.